src/main/java/ru/skypro/homework/controller/UserController.java

```diff
@@ -92,36 +86,27 @@ public ResponseEntity<UserDto> getUserInfo(@PathVariable int id) {
                     )
             }
     )
-    @PatchMapping("/me/{id}")
-    public ResponseEntity<UpdateUserDto> updateUserInfo(@PathVariable int id, @RequestBody @Valid UpdateUserDto data) {
+    @PatchMapping("/me")
+    public ResponseEntity<UpdateUserDto> updateUserInfo(@RequestBody @Valid UpdateUserDto data) {
         return ResponseEntity.ok().build();
-
     }
+
     @Operation(
             summary = "Обновление аватара авторизованного пользователя",
             responses = {
                     @ApiResponse(
                             responseCode = "200",
-                            description = "OK",
-                            content = {
-                                    @Content(
-                                            mediaType = MediaType.APPLICATION_OCTET_STREAM_VALUE
-                                    )
-                            }
+                            description = "OK"
                     ),
                     @ApiResponse(
                             responseCode = "401",
-                            description = "Unauthorized",
-                            content = {
-                                    @Content(
-                                    )
-                            }
+                            description = "Unauthorized"
                     )
 
             }
     )
-    @PatchMapping(value = "/me/avatar/{id}", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
-    public ResponseEntity<UserDto> updateAvatar(@PathVariable int id, @RequestParam MultipartFile avatar) {
+    @PatchMapping(value = "/me/image", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity<?> updateAvatar(@RequestPart MultipartFile avatar) {
```

Comment:
```suggestion
    public ResponseEntity<?> updateAvatar(@RequestPart MultipartFile("image") avatar) {
```
название части запроса - image, а не avatar

Reply:
ок